.ci/Jenkinsfile

```diff
@@ -9,6 +9,11 @@ pipeline {
     BASE_DIR = "src/github.com/elastic/${env.REPO}"
     JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
     PIPELINE_LOG_LEVEL = 'INFO'
+    DRA_OUTPUT = 'release-manager.out'
+    SLACK_CHANNEL = '#elastic-agent-control-plane'        // TODO: what slack channel
+    NOTIFY_TO = 'fleet-server+build-package@elastic.co'   // TODO: what dist liist?
```

Comment:
Who to be notified by email/slack?

Reply:
Yes, probably #elastic-agent and the elastic-agent mailing list.

Reply:
what's this? is this a valid google group? i'd rather avoid using elastic-agent mailing list to use here as it should be product specific. i'd rather use team specific or create a new one unique for shipper